predict-daily-temperature/utils.py

```diff
@@ -0,0 +1,76 @@
+import featuretools as ft
+import woodwork as ww
+import evalml
+from evalml import AutoMLSearch
+from evalml.utils import infer_feature_types
+from statsmodels.tsa.stattools import adfuller
+import matplotlib.pylab as plt
+import pandas as pd
+
+
+def test_stationarity(timeseries):
```

Comment:
👀

Reply:
oh lol I took this from this blog (https://www.analyticsvidhya.com/blog/2016/02/time-series-forecasting-codes-python/). It has a really nice graph of the rolling mean and rolling std over the time series data. But it also discussed using the Dickey-Fuller test to determine if a dataset is stationary. I don't think I ended up having any use for it here, so I'll probably remove it.